importanize/main.py

```diff
@@ -122,6 +129,23 @@ def run_importanize(path, config, args):
     text = read(path)
     file_artifacts = get_file_artifacts(path)
 
+    # Get formatter from args or importanizerc
+    if args.formatter:
+        config['formatter'] = args.formatter
+    elif config.get('formatter'):
+        pass
+    else:
+        config['formatter'] = 'IndentWithTabsFormatter'
+    # Check formatter availibility or exit
+    try:
```

Comment:
I think so. I like the idea of explicitly registering object and referring to them by them.